src/cert_context.rs

```diff
@@ -107,6 +107,47 @@ impl CertContext {
         self.get_encoded_bytes()
     }
 
+    /// Deletes the key container for both CAPI and CNG keys
+    pub fn del_key_container(&mut self) -> io::Result<()>{
```

Comment:
Leaving return type as () since by default it will succeed, if not an error will be returned.